f2/src/forms/PrivacyConsentInfoForm.js

```diff
@@ -1,109 +1,64 @@
-import { formDefaults } from './defaultValues'
-
-const validate = (values) => {
-  const errors = {}
-  //condition for an error to occur: append a lingui id to the list of error
-  if (!values.consentOptions || values.consentOptions.length < 1) {
-    errors.consentOptions = 'privacyConsentInfoForm.warning'
-  }
-
-  return errors
-}
+import { A } from '../components/formik/link'
+import { Form, Container, Row } from 'react-bootstrap'
+import { Formik, Field, ErrorMessage } from 'formik'
+import { CheckBox } from '../components/formik/checkbox'
+import { NextCancelButtons } from '../components/formik/button'
+import { Error } from '../components/formik/alert'
+import { PrivacyConsentInfoFormSchema } from './PrivacyConsentInfoFormSchema'
 
 export const PrivacyConsentInfoForm = (props) => {
-  const { i18n } = useLingui()
   const [data] = useStateValue()
   const whetherConsent = {
-    ...formDefaults.consent,
     ...data.formData.consent,
   }
-  const consentOptions = ['privacyConsentInfoForm.yes']
-  const { fyiForm } = data.formData
 
   return (
-    <React.Fragment>
-      {false ? ( // mark ids for lingui
-        <div>
-          <Trans id="privacyConsentInfoForm.yes" />
-          <Trans id="privacyConsentInfoForm.consent" />
-        </div>
-      ) : null}
-
-      <Form
-        initialValues={whetherConsent}
-        onSubmit={(values) => {
-          props.onSubmit(values)
-        }}
-        validate={validate}
-        render={({
-          handleSubmit,
-          values,
-          errors,
-          submitFailed,
-          hasValidationErrors,
-        }) => (
-          <Stack
-            as="form"
-            onSubmit={handleSubmit}
-            shouldWrapChildren
-            spacing={6}
-          >
-            {submitFailed && hasValidationErrors ? (
-              <ErrorSummary>
-                <Trans id="privacyConsentInfoForm.hasValidationErrors" />
-              </ErrorSummary>
-            ) : null}
-
-            <FormArrayControl
-              name="consentOptions"
-              errorMessage={<Trans id="privacyConsentInfoForm.warning" />}
-            >
-              {consentOptions.map((key) => {
-                return (
-                  <React.Fragment key={key}>
-                    <CheckboxAdapter name="consentOptions" value={key}>
-                      <Trans id="privacyConsentInfoForm.yes" />
-                      <A
-                        href={'/privacystatement?lang=' + i18n.locale}
-                        isExternal
-                      >
-                        <Trans id="privacyConsentInfoForm.linkOut" />
-                      </A>
-                      <Trans id="privacyConsentInfoForm.period" />
-                    </CheckboxAdapter>
-                  </React.Fragment>
-                )
-              })}
-            </FormArrayControl>
-            <NextAndCancelButtons
-              next={
-                fyiForm ? (
```

Comment:
The next page label is hard-coded. The next page is different if the user selects the FYI flow.